src/screens/FeatureList.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import Features from '../components/feature_activation/Features';
+import featureApi from '../api/featureApi';
+
+
+class FeatureList extends React.Component {
+  getFeatures = () => {
+    return featureApi.getFeatures();
```

Comment:
In practice it doesn't change the execution results, but I feel it's more trustworthy to pass the results already processed to the component instead of handling the function down and letting the component itself call it.

That way we make the communication between components only with simple serializable objects instead of complex ones.

What do you think?

Reply:
As discussed via Slack, this can stay like this for now

Reply:
This has been removed in https://github.com/HathorNetwork/hathor-explorer/pull/242/commits/2eade279971322b09009f8329d7fda60ba93b5ee